deploy/examples/external/cluster-external.yaml

```
@@ -0,0 +1,28 @@
+#################################################################################################################
+# Define the settings for the rook-ceph-external cluster with common settings for a production cluster.
+
+#   kubectl create -f ../../examples/crds.yaml -f ../../examples/common.yaml -f ../../examples/operator.yaml
+#   kubectl create -f common-external.yaml -f cluster-external.yaml
+
+#################################################################################################################
+apiVersion: ceph.rook.io/v1
+kind: CephCluster
+metadata:
+  name: rook-ceph-external
+  namespace: rook-ceph # default set to rook-ceph
```

Wherever we use the namespace we need this specific comment to easily replace according to [this topic](https://rook.io/docs/rook/latest/Storage-Configuration/Advanced/ceph-configuration/#using-alternate-namespaces)
```suggestion
  namespace: rook-ceph # namespace:cluster
```